server/server.go

```diff
@@ -94,6 +94,7 @@ func (server *GripServer) Serve(pctx context.Context) error {
 				streamErrorInterceptor(),
 			),
 		),
+		grpc.MaxMsgSize(1024*1028*16),
```

Comment:
This option is depreciated. You should use https://godoc.org/google.golang.org/grpc#MaxCallSendMsgSize